### ISSUE TITLE
Create an "unstable" release?

### ISSUE THREAD

**Issue by danielshahaf** [zsh-users/zsh-syntax-highlighting#711]
There's been a large number of small changes in HEAD:

https://github.com/zsh-users/zsh-syntax-highlighting/blob/aecfd61bbda9b8739c48e7a2818438cb0f134791/changelog.md#changes-in-head

I think it's premature to call them stable — they could have functionality or performance bugs — but I'd like to let them get more testing.  So, how about adopting creating an "unstable" release off master?  We can call it "alpha1", or adopt the even-odd minor number convention, or whatever else; the point would be to create a tarball to encourage testing by downstreams, without yet declaring the work stable.

Since changelog is already written (I've been writing it as I went), the work involved in making this happen would be minimal; it'll be just tagging and bumping the version number on master.

**Comment by phy1729** [comment]
I don't understand the point. We already have testing of master via oh-my-zsh updating users automatically (by default if memory serves). Package maintainers can already package master if they so desire, and it's clear to everyone there are no API/backwards-compat promises. If we have unstable releases, the chance of getting somewhere that should be stable with backwards compat promises increases.

**Comment by danielshahaf** [comment]
Fair point about omz.

I thought that if we made an unstable release we could get even more testing.  E.g., in Debian there's the `experimental` suite.  If a z-sy-h snapshot was updated there, that should make it easier for users to use a bleeding edge z-sy-h (even on systems that run `stable`, if they wished).

I don't think there's anything stopping me from packaging snapshots of HEAD into `experimental`, but I don't think that'd be appropriate.  There's a difference between a random git snapshot and an odd-minor-number release, after all.

On the other hand, I see your point: we may well have enough testing as it is thanks to users of master.
